Autopilot/Inc/gimbal.hpp

```diff
@@ -0,0 +1,10 @@
+/**
+ * Gimbal command processing
+ * Author: Anthony Luo (a26luo@uwaterloo.ca)
+ */
+
+#ifndef GIMBAL_HPP
+#define GIMBAL_HPP
+
+float getPitchPercent();
+float getYawPercent();
```

Comment:
```cpp
/**
*
*/
```

Function comments needed. 

Also, need some newlines at the end of the file. (hit enter a few times)